cmake/OpenCVFindVA_INTEL.cmake

```diff
@@ -5,7 +5,7 @@
 # VA_INTEL_IOCL_ROOT - root of Intel OCL installation
 
 if(UNIX AND NOT ANDROID)
-    if($ENV{VA_INTEL_IOCL_ROOT})
+    if(DEFINED ENV{VA_INTEL_IOCL_ROOT})
         set(VA_INTEL_IOCL_ROOT $ENV{VA_INTEL_IOCL_ROOT})
     else()
         set(VA_INTEL_IOCL_ROOT "/opt/intel/opencl")
```

Comment:
Also this code breaks priority of sources for variable values:
- cmake default ("/opt/intel/opencl" - lowest)
- environment
- cmake variable in command line (high priority)

So, please use this code snippet:
```
    ocv_check_environment_variables(VA_INTEL_IOCL_ROOT)
    if(NOT DEFINED VA_INTEL_IOCL_ROOT)
      set(VA_INTEL_IOCL_ROOT "/opt/intel/opencl")
    endif()
```